tests/tabular/test_dummy_model.py

```diff
@@ -64,11 +59,49 @@ def test_roc_condition_ratio_more_than_passed(iris_clean):
 
     assert_that(result, has_items(
         equal_condition_result(is_pass=True,
-                               details='All classes passed, minimum AUC found is 0.71 for class 1',
+                               details='All classes passed, minimum AUC found is 1 for class 2',
```

Comment:
Why did the result change?

Reply:
Replaced LogReg model with the default iris model (which is better)